src/backend/aspen/workflows/nextstrain_run/run_nextstrain_sc2.sh

```diff
@@ -57,6 +57,12 @@ aligned_gisaid_location=$(
 )
 
 
+# Tree builds break if include.txt is empty. If we don't have any selected samples,
+# then copy in nextstrain's defaults.
+if [ ! -s /ncov/data/include.txt ]; then
+    cp /ncov/defaults/include.txt /ncov/data/include.txt
+fi
+
 # Persist the build config we generated.
 $aws s3 cp /ncov/my_profiles/aspen/builds.yaml "${s3_prefix}/builds.yaml"
 $aws s3 cp /ncov/data/include.txt "${s3_prefix}/include.txt"
```

Comment:
how come this `cp` command hasn't failed so far when this `data/include.txt` doesn't exist (like for all automatic trees)?

Reply:
the file always exists, it's just empty sometimes!